resources/function_help/json/is_selected

```diff
@@ -14,13 +14,18 @@
             "variant": "One 'feature' parameter",
             "variant_description": "If called with a 'feature' parameter only, the function returns true if the specified feature from the current layer is selected.",
             "arguments": [ { "arg": "feature", "description": "The feature which should be checked for selection." } ],
-            "examples": [ { "expression": "is_selected(@atlas_feature)", "returns": "True if the current atlas feature is selected." } ]
+            "examples": [ { "expression": "is_selected(@atlas_feature)", "returns": "True if the current atlas feature is selected." },
```

Comment:
I had completly missed the previous PR but I think it would be a good idea to mention that the one using the atlas_feature only will only work if used in the layer that is used as an atlas (and if the expression gets a valid atlas context scope at eval time).